pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -1229,7 +1229,7 @@ func TestNodesWherePreemptionMightHelp(t *testing.T) {
 				"machine1": []algorithm.PredicateFailureReason{algorithmpredicates.ErrNodeUnderDiskPressure},
 				"machine2": []algorithm.PredicateFailureReason{algorithmpredicates.ErrNodeUnderPIDPressure},
 				"machine3": []algorithm.PredicateFailureReason{algorithmpredicates.ErrNodeUnderMemoryPressure},
-				"machine4": []algorithm.PredicateFailureReason{algorithmpredicates.ErrNodeOutOfDisk},
+				"machine4": []algorithm.PredicateFailureReason{algorithmpredicates.ErrNodeUnderDiskPressure},
```

Comment:
`machine1` is already tested for disk pressure.

Reply:
Good catch, thanks!